packages/office-ui-fabric-react/src/components/__snapshots__/Pivot.SeparateNoSelectedKey.Example.tsx.shot

```diff
@@ -31,511 +31,524 @@ exports[`Component Examples renders Pivot.SeparateNoSelectedKey.Example.tsx corr
       <div
```

Comment:
Why is this snapshot being updated?

Reply:
Not entirely sure why that happend. I pulled from upstream and reverted the snapshot update and it works now.